### ISSUE TITLE
Refused to display 'https://login.microsoftonline.com/' in a frame  because it set 'X-Frame-Options' to 'deny'

### ISSUE THREAD

**Issue by webelieve** [AzureAD/microsoft-authentication-library-for-js#7458]
### Core Library

MSAL.js (@azure/msal-browser)

### Core Library Version

 "react": "^18.2.0",

### Wrapper Library

MSAL React (@azure/msal-react)

### Wrapper Library Version

 "react": "^18.2.0",

### Public or Confidential Client?

Public

### Description

Hi! 

We have encountered issue on this error - Refused to display 'https://login.microsoftonline.com/' in a frame 
because it set 'X-Frame-Options' to 'deny'. Could you please advise? Thank you!

When users logged in with multiple accounts https://myaccount.microsoft.com/, then try to access to our app.

Results:
- Console log tab: Refused to display 'https://login.microsoftonline.com/' in a frame because it set 'X-Frame-Options' to 'deny'

Copy https://login.microsoftonline.com in a new tab, it takes the user to 
- [office.com ](https://www.office.com/?auth=2)

### MSAL Configuration

```javascript
export const msalConfig: Msal.Configuration = {
  auth: {
    clientId: EnvironmentService.getAzureClientId(),
    redirectUri: `${window.location.origin}`,
    authority: `https://login.microsoftonline.com/getAzureTenantId()`
  },
  cache: {
    cacheLocation: "sessionStorage",
    storeAuthStateInCookie: true,
  }
}
```

**Comment by webelieve** [comment]
@tnorling Hi Thomas, I noticed that you answered a previous post on this issue. I appreciate if you can assist. Thank you!

https://github.com/AzureAD/microsoft-authentication-library-for-js/issues/2731 

We are dealing with a scenario where **multiple** accounts are logged in **simultaneously**. How should we handle this? We want to authenticate the user after they select the their primary account and access our app, ensuring they see the same content as they only logged in using primary accounts. However, we are currently facing an issue where, after the user selecting their primary account from Azure logging popup window, our app encounters a “**Refused to display 'https://login.microsoftonline.com/' in a frame because it set 'X-Frame-Options' to 'deny**” error in the console log, though the user will not see unless they use DevTools to inspect. Is it possible to allow the user to log in **silently** even they logged by multiple accounts?

Please advise. Thanks!

**Comment by tnorling** [comment]
> Is it possible to allow the user to log in silently even they logged by multiple accounts?

Yes but you need to provide a login_hint to tell the server which account to use. See [here](https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/dev/lib/msal-browser/docs/login-user.md#silent-login-with-ssosilent) for more information about hints. This applies to both ssoSilent and acquireTokenSilent.

In general X-Frame-Options Deny is thrown when the identity provider is attempting to show some information inside an iframe, either an error or some sort of prompt, which is not allowed. If you need more support I suggest opening a [support ticket](https://azure.microsoft.com/en-us/support/create-ticket/)
